package.json

```diff
@@ -11,14 +11,16 @@
     "scripts": {
         "postinstall": "pnpm build",
 
-        "build": "pnpm build:eslint && pnpm build:tsup && pnpm --filter ./packages/* -r build",
+        "build": "pnpm build:tsup && pnpm build:swc && pnpm build:eslint && pnpm --filter ./packages/* -r build",
```

Comment:
ça va tu buildé 2 fois swc tsup et eslint?

Reply:
Probablement, ça ne me dérangeait pas tant cependant car ce sont des petits projets.

Si tu as une proposition, je suis ouvert!

Reply:
Fix with a wonderful: `pnpm build:tsup && pnpm build:swc && pnpm build:eslint && pnpm build:stylelint && pnpm build:typescript && pnpm build:browserlist && pnpm build:postcss` to garantee the execution order.